ui/desktop/tests/acceptance/scopes-test.js

```diff
@@ -180,7 +180,16 @@ module('Acceptance | scopes', function (hooks) {
 
   test('can navigate among org scopes via header navigation', async function (assert) {
     assert.expect(3);
-    this.stubClientDaemonSearch('targets', 'targets', 'targets', 'targets');
+    this.stubClientDaemonSearch(
+      'targets',
+      'targets',
+      'targets',
+      'targets',
+      'targets',
+      'targets',
+      'targets',
+      'targets',
```

Comment:
Can you provide some context on why we do this?

Reply:
Absolutely. Each `'targets'` in this stub represents a query to the client daemon to fetch targets. In this specific tests, we query targets eight times.  Once when we do the initial visit on line 193. Then three more times with each click in the test. We could pull each click and assert into their own tests but it would still need four `'targets'` for each test. Does that help explain why this is needed?